src/fetch.ts

```diff
@@ -0,0 +1,4 @@
+import * as nodeFetch from 'node-fetch';
+export default typeof window !== 'undefined'
+  ? window.fetch.bind(window)
+  : ((nodeFetch as unknown) as typeof fetch);
```

Comment:
Nice man! I love this! Sorry, I'm bored and reading your code while I watch election results on TV

Reply:
Yeah this one took me a bit to figure out hahaha.